app/routes/capture.js

```diff
@@ -40,6 +41,12 @@ export default Ember.Route.extend({
         );
     },
 
+    afterModel(captureModel) {
+        if (electronUtils.isElectron()) {
+            electronUtils.addRecentDocument(captureModel.filePath);
```

Comment:
Would it make sense to add the file to "recent files" after the file is analyzed? This way, you would avoid adding files that for one reason or another don't properly work.